### DISCUSSION TITLE
Could not found authentication with username password

### DISCUSSION THREAD

**Discussion by Touchteng** [spatie/ssh#100] [Q&A]
Hi expert,

I want to access to Linux server via SSH. I could not found how can I configure password. Please help me!

Thanks,

**Comment by cornzzy** [comment]
This package doesn't support the most common and simple form auth of ssh [because he doesn't need it](https://github.com/spatie/ssh/issues/36#issuecomment-698215014).

**Reply by Touchteng** [reply]
Hi @cornzzy,
May you elaborate on this matter. I have only username and password for access via SSH. How can I configure in Laravel in order to make it works?

**Reply by cornzzy** [reply]
just use something else man. here is a popular ssh lib:
https://github.com/phpseclib/phpseclib

**Comment by sandeshjangam** [comment]
Feel free to use this: https://github.com/sandeshjangam/tiny-ssh-php
